.travis.yml

```diff
@@ -5,9 +5,21 @@ language: python
 matrix:
     include:
         - python: 2.7
-          env: KERAS_BACKEND=tensorflow
+          env:
+            - KERAS_BACKEND=tensorflow
+            - TENSORFLOW_V=1.0.1
```

Comment:
We should also test tf versions 1.1 and 1.2, right? We don't need to get all combinations of python version and tensorflow version, but I think we should get all supported minor versions of tensorflow.